model/task_lifecycle.go

```diff
@@ -790,7 +789,17 @@ func updateDisplayTaskAndCache(t *task.Task) error {
 	if err != nil {
 		return errors.Wrap(err, "error updating display task")
 	}
-	return build.UpdateCachedTask(t.DisplayTask, 0)
+	err = build.UpdateCachedTask(t.DisplayTask, 0)
+	if err != nil {
```

Comment:
no need for an if statement here, since `grip.Errors(message.WrapError(nil))` is a noop